python/opae.admin/pyproject.toml

```diff
@@ -23,33 +23,42 @@
 # CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-from setuptools import setup, find_namespace_packages
 
-setup(
-    name="opae.admin",
-    version='1.4.3',
-    packages=find_namespace_packages(include=['opae.*']),
-    entry_points={
-        'console_scripts': [
-            'fpgasupdate = opae.admin.tools.fpgasupdate:main',
-            'rsu = opae.admin.tools.rsu:main',
-            'super-rsu = opae.admin.tools.super_rsu:main',
-            'fpgaflash = opae.admin.tools.fpgaflash:main',
-            'fpgaotsu = opae.admin.tools.fpgaotsu:main',
-            'fpgaport = opae.admin.tools.fpgaport:main',
-            'bitstreaminfo = opae.admin.tools.bitstream_info:main',
-            'opaevfio = opae.admin.tools.opaevfio:main',
-            'pci_device = opae.admin.tools.pci_device:main',
-            'regmap-debugfs = opae.admin.tools.regmap_debugfs:main',
-            'fpgareg = opae.admin.tools.fpgareg:main',
-        ]
-    },
-    install_requires=[],
-    description="opae.admin provides Python classes for interfacing with"
-                "OPAE kernel drivers",
-    license="BSD3",
-    keywords="OPAE accelerator fpga kernel sysfs",
-    data_files=[('share/doc/opae.admin',
-                 ['LICENSE'])],
-    url="https://opae.github.io",
-)
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "opae.admin"
+version = "1.4.4"
+description = "opae.admin provides Python classes for interfacing with OPAE kernel drivers"
+#readme = ""
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.6"
+
+[tool.setuptools]
+packages = [
+"opae",
+"opae.admin",
+"opae.admin.tools",
+"opae.admin.utils",
+]
+
+[tool.setuptools.package-data]
+"*" = ["LICENSE"]
+
+[project.scripts]
+fpgasupdate = "opae.admin.tools.fpgasupdate:main"
+rsu = "opae.admin.tools.rsu:main"
+super-rsu = "opae.admin.tools.super_rsu:main"
+fpgaflash = "opae.admin.tools.fpgaflash:main"
+fpgaotsu = "opae.admin.tools.fpgaotsu:main"
+fpgaport = "opae.admin.tools.fpgaport:main"
+bitstreaminfo = "opae.admin.tools.bitstream_info:main"
+opaevfio = "opae.admin.tools.opaevfio:main"
+pci_device = "opae.admin.tools.pci_device:main"
+regmap-debugfs = "opae.admin.tools.regmap_debugfs:main"
+fpgareg = "opae.admin.tools.fpgareg:main"
+
+[project.urls]
+Homepage = "https://opae.github.io"
```

Comment:
"https://opae.github.io -> ofs.github.io

Reply:
Let's change the URL in a separate pull request.  That is outside the scope of this change.